packages/components/src/sortTabs/SortTab.js

```diff
@@ -110,18 +110,22 @@ class SortTab extends React.Component<Props> {
 export default withHover(SortTab);
 
 const borderRadius = parseInt(defaultTokens.borderRadiusSmall, 10);
-const containerHeight = 75;
+const containerHeight = {
```

Comment:
It definitely looks better. But I'm just slightly worried about size on smaller screens, where there is not much space left for results.

<img width="379" alt="Screenshot 2019-05-02 at 11 36 24" src="https://user-images.githubusercontent.com/2660330/57067586-bc053a00-6ccf-11e9-8f1a-13742c413e7e.png">

What about reducing content to just one line and remove unnecessary empty space. Something like.

<img width="378" alt="Screenshot 2019-05-02 at 11 39 59" src="https://user-images.githubusercontent.com/2660330/57067688-fa025e00-6ccf-11e9-9ec4-efe3eb905ddf.png">

Or maybe even better, we can hide this content into badge which will open options inside modal, like in current mobile app. But we can probably save this iteration for later when/if we will have more filters.

![Screenshot_20190502-114845](https://user-images.githubusercontent.com/2660330/57068003-cbd14e00-6cd0-11e9-88eb-d96642b12725.png)